src/components/FontAwesomeIcon.js

```diff
@@ -12,6 +12,7 @@ const defaultProps = {
   className: '',
   mask: null,
   maskId: null,
+  // the fixedWidth property has been deprecated as of version 7
```

Comment:
I don't believe we do anything additional here with fixedWidth, rotateBy, and widthAuto.
This is the `defaultProps {}`